selenpy/support/browser.py

```diff
@@ -43,3 +47,13 @@ def wait_until(webdriver_condition, timeout=None, polling=None):
         polling = config.poll_during_waits
 
     return wait_for(get_driver(), webdriver_condition, timeout, polling)
+
+def get_alert():
+    try:
+        WebDriverWait(get_driver(),config.timeout).until(EC.alert_is_present())
+        return get_driver().switch_to.alert
+    except UnexpectedAlertPresentException:
+        logging.info("no alert")
```

Comment:
What's happened when the exception is occurred? just log ? is it a returned function?

Reply:
I updated